.circleci/config.yml

```diff
@@ -31,6 +31,7 @@ jobs:
       - image: *GOLANG_IMAGE
     steps:
       - checkout
+      - run: sudo apt-get install -y bzr
```

Comment:
@alvin-huang do we want to do this here? I think we should probably add this to the make tools instead right?

Also, I wonder if the replace directive I asked @likexian to remove which redirects off launchpad.net would also fix this. Which is worse having every contributor need `bzr` or maintaining a redirect in go mod?

When we get to 1.13 and use a module proxy this will also go away again...

Reply:
I think maintaining the redirect is better than having others install `bzr`. I don't see that line changing much between now and when we get to 1.13 so it's minimal maintenance.

Reply:
@banks @alvin-huang 
So, what can I do for this PR to go on?

Reply:
Apologies Li, 

We've been waiting to get consensus on the best approach here before we ask you to change anything else again as it's not ideal to keep changing our minds!

US folks are out this week but I'll try to get back to you next week so we can move this on!

Thanks again for you contribution.

Reply:
Thanks for the reply, If there is something I can do, it is my pleaseure.